### PR TITLE
chore: 3.7.0 CHANGELOG.md

### PR DESCRIPTION
**Pull request by alilleybrinker**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,115 @@
...
+have any built in data sources or analyses, and to instead let `hc` be the
+coordination and execution system which delegates the actual data collection
+and analysis to plugins exclusively.
+
+* created plugin version of github api queries by [@j-lanson](https://github.com/j-lanson) in [#475](https://github.com/mitre/hipcheck/pull/475)
+
+### Code Cleanup
+
+The changes to introduce the plugin system left a lot of cruft on the table to
+be removed, which we're not working on doing.
```
[the remark this answers is not in the document]
Comment (alilleybrinker) [reply CHANGELOG.md:82]: Good catch.

**Comment by alilleybrinker** [comment]
Note that this also includes fixes to work with the latest version of `git-cliff`.

File: CHANGELOG.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (j-lanson) [reply CHANGELOG.md:82]: Still doesn't appear fixed

[the remark this answers is not in the document]
Comment (alilleybrinker) [reply CHANGELOG.md]: Ugh, forgot `git add`. Should be fixed now.

**Pull request merged**